features/pageobjects/card1Form.js

```diff
@@ -0,0 +1,48 @@
+const { $ } = require("@wdio/globals");
+const BasePage = require("./baseForm");
+const BaseElement = require("./baseElement");
+
+class card1Page extends BasePage {
+  constructor() {
+    super(".dropdown__field");
+
+    this.TermsAndConditions = new BaseElement(".icon-check");
+    this.domainDropDown = new BaseElement(
+      '//span[@class="icon icon-chevron-down"]'
+    );
+    this.gov = new BaseElement(
+      "/html/body/div/div[1]/div[2]/div[4]/div/div[1]/div/div[3]/form/div[1]/div[3]/div[4]/div/div[2]/div[5]"

```

Comment:
This locator is very unreliable, even if the minor changes in structure will happen it won't be working